### ISSUE TITLE
FYI fails when doing multiple requests in parallel

### ISSUE THREAD

**Issue by panique** [alephdata/convert-document#17]
Just as an information: This fails when doing multiple requests in parallel

**Comment by pudo** [comment]
Yep, it's doing a lot of locking to make sure that only one request is being processed at the same time. The intended outcome is for the server to return an error 503 when the client should back off and try again, 400 if the document itself is invalid.
